README.md

```
@@ -58,17 +58,22 @@ source install/setup.bash
 
 Simulation:
 
 ```bash
-ros2 launch husarion_ugv_gazebo simulation.launch.py
+ros2 launch husarion_ugv_gazebo simulation.launch.yaml
 ```
 
 > [!IMPORTANT]
-> You can change spawning robot in simulation, by adding `robot_model:={robot_model}` argument.
+> You can change which robot is spawned in simulation by passing the `robot_model:={robot_model}` argument.
+
```

markdownlint-cli2 (0.22.1)

[warning] Blank line inside blockquote

(MD028, no-blanks-blockquote)